WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/WPScreenshotTest.java

```diff
@@ -75,6 +76,15 @@ public void wPScreenshotTest() {
         logoutIfNecessary();
     }
 
+    private void navigateMySite() {
+        // Click on the "Sites" tab and take a screenshot
+        clickOn(R.id.nav_sites);
+
+        waitForElementToBeDisplayedWithoutFailure(R.id.row_blog_posts);
+
+        takeScreenshot("4-screenshot-my-site");
```

Comment:
Aha. Yes, I meant to keep the names matching the previous pattern for this edit and then update them all at once later with better names matching what's in each screenshot instead of the heading later so that change happens all in one go.